### PR TITLE
Modified the docker SP metadata to just have one entry with EntitiesDescriptor

### PR DESCRIPTION
**Pull request by blen-desta**
#### What does this PR do (if it's not clear from the Title)? Please include any specification references that apply.
- Modified the docker SP metadata to just have one entry with `EntitiesDescriptor`.
- Addressed warning

#### Checklist:
- [ ] SAML Spec Table of Contents documentation updated
- [ ] Unit Tests Added/Modified

### CODE REVIEW

**Comment by blen-desta** [comment]
build now

File: deployment/docker/etc/org.codice.ddf.security.idp.server.IdpEndpoint.config [hunk 1]
```diff
@@ -3,8 +3,7 @@ guestAccess=B"true"
...
+  ]
```
Comment (brjeter) [inline deployment/docker/etc/org.codice.ddf.security.idp.server.IdpEndpoint.config]: 💯

**Comment by blen-desta** [comment]
Connection timeout failure
build now

**Comment by blen-desta** [comment]
build now

**Comment by blen-desta** [comment]
build now

**Pull request merged**